build_tools/ci/cpu_comparison/output_comparer.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import sys
+
+
+def compare(npy_cpu1, npy_aie1):
+
+    if npy_cpu1.shape != npy_aie1.shape:
+        raise ValueError(
+            "The two outputs have different shapes: {} and {}".format(
+                npy_cpu1.shape, npy_aie1.shape
+            )
+        )
+
+    are_close = np.allclose(npy_cpu1, npy_aie1, rtol=1e-3, atol=1e-3)
```

Comment:
rtol and atol tolerance values seem to be large for float numbers? (I'm not sure what values are appropriate).

Reply:
It's context specific, there are cases where 1e-3 is probably too low. I'll try and make it a command line arg